godot-kotlin/godot-library/src/nativeCore/kotlin/godot/core/type/Basis.kt

```diff
@@ -762,6 +795,72 @@ class Basis(
         result = 31 * result + _z.hashCode()
         return result
     }
+
+
+    fun set(xAxis: Vector3, yAxis: Vector3, zAxis: Vector3) {
+        setAxis(0, xAxis)
+        setAxis(1, yAxis)
+        setAxis(2, zAxis)
+    }
+
+    /*
+     * GDScript related members
+     */
+    constructor(xAxis: Vector3, yAxis: Vector3, zAxis: Vector3) : this() {
```

Comment:
Basis is exposed as column major in GDScript but internally uses a row major format